Description: Fix minor man page error
 the return type for nids_register_chksum_ctl is void and not int, update the
 man apge accordingly
Bug: https://sourceforge.net/p/libnids/patches/7/
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/src/libnids.3
+++ b/src/libnids.3
@@ -55,7 +55,7 @@
 int
 \fBnids_getfd\fR(void);
 
-int
+void
 \fBnids_register_chksum_ctl\fR(struct nids_chksum_ctl *, int);
 
 void
